lib/hsa/mcwamp_hsa.cpp

```diff
@@ -2799,6 +2800,9 @@ class HSAContext final : public KalmarContext
     */
     hsa_agent_t host;
 
+    cxlProfiler profiler;
+    cxlMarker contextMarker = CXL_MARKER_CLASS;
```

Comment:
@otauqir since the dependency to `hc_profile` is directly imposed here, why wouldn't `hc_profile` be simply placed inside `mcwamp_hsa` library, but to make it a standalone one, and has to be linked by `hcc-config --ldflags`? It could potentially make HCC applications more fragile, as *not* every HCC application are linked using `hcc-config`.